tests/unit/ui/components/TestFormOverlay.js

```diff
@@ -33,23 +33,29 @@ describe(__filename, () => {
     );
   };
 
+  const renderOpen = (customProps = {}) => {
+    const id = customProps.id || 'example-form-id';
+    store.dispatch(openFormOverlay(id));
+    return render({ id, ...customProps });
+  };
+
   it('renders a custom class name', () => {
-    const root = render({ className: 'some-class' });
+    const root = renderOpen({ className: 'some-class' });
 
     expect(root).toHaveClassName('some-class');
     expect(root).toHaveClassName('FormOverlay');
   });
 
   it('renders a title', () => {
-    const root = render({ title: 'Some Form' });
+    const root = renderOpen({ title: 'Some Form' });
 
     expect(root.find('.FormOverlay-h3')).toHaveText('Some Form');
   });
 
-  it('is open by default', () => {
+  it('is closed by default', () => {
     const root = render({ children: [<input key={1} />] });
 
-    expect(root.find('input')).toHaveLength(1);
+    expect(root.find('input')).toHaveLength(0);
```

Comment:
Checking for `.FormOverlay` seems more accurate; it's likely there will be an input but we should check for the entire element. Sorry, I know this is mostly from a previous patch but this tweak made me notice it more 😄